File: Spring/HostelManagement/src/main/java/in/ac/auttvl/hostel/model/Warden.java
package in.ac.auttvl.hostel.model;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Warden {
	
	@Id
	@GeneratedValue(strategy = GenerationType.AUTO)
	@Column(length =100)
	private int id;
	private int orderNo;
	private int wardenId;
	private String name;
	private String email;
	private String gender;
	private String dateOfBirth;
	private String phoneNo;
	private String password;
	private String confirmPassword;
	private String photo;
	private String photoPath;
	private String updatedBy;
	private String updatedOn;
	
	
	public Warden(int id, int orderNo, int wardenId, String name, String email, String gender, String dateOfBirth,
			String phoneNo, String password, String confirmPassword, String photo, String photoPath, String updatedBy,
			String updatedOn) {
		super();
		this.id = id;
		this.orderNo = orderNo;
		this.wardenId = wardenId;
		this.name = name;
		this.email = email;
		this.gender = gender;
		this.dateOfBirth = dateOfBirth;
		this.phoneNo = phoneNo;
		this.password = password;
		this.confirmPassword = confirmPassword;
		this.photo = photo;
		this.photoPath = photoPath;
		this.updatedBy = updatedBy;
		this.updatedOn = updatedOn;
	}


	public Warden() {
		super();
	}


	public int getId() {
		return id;
	}


	public void setId(int id) {
		this.id = id;
	}


	public int getOrderNo() {
		return orderNo;
	}


	public void setOrderNo(int orderNo) {
		this.orderNo = orderNo;
	}


	public int getWardenId() {
		return wardenId;
	}


	public void setWardenId(int wardenId) {
		this.wardenId = wardenId;
	}


	public String getName() {
		return name;
	}


	public void setName(String name) {
		this.name = name;
	}


	public String getEmail() {
		return email;
	}


	public void setEmail(String email) {
		this.email = email;
	}


	public String getGender() {
		return gender;
	}


	public void setGender(String gender) {
		this.gender = gender;
	}


	public String getDateOfBirth() {
		return dateOfBirth;
	}


	public void setDateOfBirth(String dateOfBirth) {
		this.dateOfBirth = dateOfBirth;
	}


	public String getPhoneNo() {
		return phoneNo;
	}


	public void setPhoneNo(String phoneNo) {
		this.phoneNo = phoneNo;
	}


	public String getPassword() {
		return password;
	}


	public void setPassword(String password) {
		this.password = password;
	}


	public String getConfirmPassword() {
		return confirmPassword;
	}


	public void setConfirmPassword(String confirmPassword) {
		this.confirmPassword = confirmPassword;
	}


	public String getPhoto() {
		return photo;
	}


	public void setPhoto(String photo) {
		this.photo = photo;
	}


	public String getPhotoPath() {
		return photoPath;
	}


	public void setPhotoPath(String photoPath) {
		this.photoPath = photoPath;
	}


	public String getUpdatedBy() {
		return updatedBy;
	}


	public void setUpdatedBy(String updatedBy) {
		this.updatedBy = updatedBy;
	}


	public String getUpdatedOn() {
		return updatedOn;
	}


	public void setUpdatedOn(String updatedOn) {
		this.updatedOn = updatedOn;
	}
	
	
}

File: Angular/HostelManagement/src/app/model/warden.ts
export class Warden {
  id : number;
  name : string;
  gender : string;
  dateOfBirth : string;
  age : number;
  bloodGrp : string;

  wardenId : string;
  phoneNo : string;
  email : string;
  password : string;
  confirmPassword : string;

  status : string;
  hostel : string;

  //address
  street: string;
  city: string;
  district: string;
  state: string;
  pincode: string;

  image : string;
  imagePath : string;
  updatedBy : string;
  updatedOn : string;
  dateOfEnrollment : string;

  verify : string;


  constructor(){
    this.id = 0;
    this.name = '';
    this.gender = '';
    this.dateOfBirth = '';
    this.age = 0;
    this.bloodGrp = '';

    this.wardenId = '';
    this.phoneNo = '';
    this.email = '';
    this.password = '';
    this.confirmPassword = '';

    this.status = '';
    this.hostel = '';

    this.street = '';
    this.city = '';
    this.district = '';
    this.state = '';
    this.pincode = '';

    this.image = '';
    this.imagePath = '';
    this.updatedBy = '';
    this.updatedOn = '';
    this.dateOfEnrollment = '';

    this.verify = '';
  }
}
